src/boot/efi/boot.c

```diff
@@ -21,6 +21,8 @@
 #define EFI_OS_INDICATIONS_BOOT_TO_FW_UI 0x0000000000000001ULL
 #endif
 
+#define TIMEOUT_UNSET (UINT32_MAX - 1U)
```

Comment:
maybe also add `TIMEOUT_FORCE_MENU` as UINT32_MAX?

and maybe TIMEOUT_MENU_OFF?